Mathlib/RingTheory/PowerSeries/Restricted.lean

```diff
@@ -0,0 +1,227 @@
+import Mathlib.Analysis.Normed.Group.Ultra
```

Comment:
You'll need to add a file header here - [see here for how to set things up](https://leanprover-community.github.io/contribute/style.html#header-and-imports)